src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
+
+const table = document.querySelector('.dashboard');
+
+function createCell(content) {
+  const cell = document.createElement('td');
+  cell.textContent = content;
+  return cell;
+}
+
+function calculateAge(person) {
+  return person.died - person.born;
+}
```

Comment:
The `calculateAge` function is simple and clear. However, the function name could be more descriptive. Consider renaming it to something like `calculatePersonAge` or `getPersonAge` to make it clear that it calculates the age of a person.